homeassistant/components/keymitt_ble/entity.py

```diff
@@ -0,0 +1,38 @@
+"""MicroBot class."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.bluetooth.passive_update_coordinator import (
+    PassiveBluetoothCoordinatorEntity,
+)
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.entity import DeviceInfo
+
+from .const import MANUFACTURER
+
+
+class MicroBotEntity(PassiveBluetoothCoordinatorEntity):
+    """Generic entity for all MicroBots."""
+
+    def __init__(self, coordinator, config_entry):
+        """Initialise the entity."""
+        super().__init__(coordinator)
+        self._address = self.coordinator.ble_device.address
+        self._attr_name = "MicroBot Push"
```

Comment:
It would be nice to opt into using entity name instead
https://developers.home-assistant.io/blog/2022/07/10/entity_naming